tests/test_qc_metrics.py

```diff
@@ -198,8 +201,18 @@ def adata_mito():
     return adata_dense, init_var
 
 
+skip_if_adata_0_11_4 = pytest.mark.skipif(
+    Version(version("anndata")) >= Version("0.11.4.dev2"),
+    reason="Old AnnData doesn’t have sparse test helpers",
+)
```

Comment:
Isn't the reason that we are removing support in 0.12, not that we lost a test helper in 0.11.4?

Reply:
https://github.com/scverse/anndata/pull/1829

Reply:
```suggestion
skip_if_adata_0_11_4 = pytest.mark.skipif(
    Version(version("anndata")) >= Version("0.11.4.dev2"),
    reason="Newer AnnData removes implicit support for COO matrices",
)
```